client/src/utils/QueryFieldsUtil.ts

```diff
@@ -219,6 +219,13 @@ const QUERY_FIELDS_MAP: QueryFieldMap = {
     baseType: 'string',
     lexType: 'string',
   },
+  // MODELS
+  MODELS_TEXT_SEARCH: {
+    ..._field('modelsTextSearch', 'Text Search'),
```

Comment:
I think this could be just `Keyword` to be consistent with how we do it in the Knowledge space. From Lex also says `Choose a field to search` so I think it makes more sense.